pkg/providers/instancetype/types.go

```diff
@@ -46,12 +46,15 @@ const (
 
 var (
 	instanceTypeScheme = regexp.MustCompile(`(^[a-z]+)(\-[0-9]+tb)?([0-9]+).*\.`)
+
+	instanceTypesSupportedForWindows = []string{"m4.16xlarge"}
```

Comment:
Windows support other instance types as well in addition to `m4.16xlarge`.

An exhaustive list would be https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/master/pkg/aws/vpc/limits.go.

I see the documentation is behind- https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html. Let me check with the team to test and get this part rectified.

Reply:
Wait for a clear rule to refine this getOS code